src/main/webapp/app/feature-overview/feature-overview.scss

```diff
@@ -0,0 +1,264 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu&display=swap');
```

Comment:
Do we need to load external fonts? What about using the same font setup as elsewhere?

Reply:
I agree, please use the same font as in all other places.
It's really `bad practice` to link to external URLs like this

Reply:
see 6d76e00